SIGNALESP/cc1101.h

```diff
@@ -506,6 +537,18 @@ namespace cc1101 {
 		setReceiveMode();
 	}
 
+#ifdef _CC1101_DEBUG_CONFIG
+  void dumpConfigRegister() {
+    Serial.printf("\ndump config register:\n");
```

Comment:
Eigentlich sollten Debug ausgaben in DBG_PRINT erfolgen.

Mittels ETHERNET_DEBUG kann man dann festlegen ob seriell oder via ethernet die Debug Ausgaben erfolgen sollen

Reply:
ist nur Entwicklungskram (EEPROM vs. Config Register) und mit Compiler-Flag versehen. Werde ich ausschalten.